package_build_list_host_linux.json

```diff
@@ -5,7 +5,8 @@
     "comment4" : "Note:  Build from source occurs before build_from_folder",
     "build_from_source": {
         "assimp-5.2.5-rev1-linux":  "Scripts/extras/pull_and_build_from_git.py ../../package-system/assimp --platform-name Linux --package-root ../../package-system --clean",
-        "AWSGameLiftServerSDK-5.0.0-rev1-linux-openssl-1": "package-system/AWSGameLiftServerSDK/build_package_image.py --platform-name linux",
+        "AWSGameLiftServerSDK-5.0.0-rev1-linux-aarch64": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSGameLiftServerSDK --platform-name Linux-aarch64 --clean",
```

Comment:
Are the aarch64 entries needed in `package_build_list_host_linux.json`?

Reply:
This allows for cross-compilation of aarch64 on an x86_64 machine

Reply:
Ok thanks for the clarification